### PR TITLE
new->failed transition & wait for PAC timer if > 0 local candidates

### PR DESCRIPTION
**Pull request by jan-ivar**
Fixes https://github.com/w3c/webrtc-pc/issues/2698.

***
<a href="https://pr-preview.s3.amazonaws.com/jan-ivar/webrtc-pc/pull/2704.html" title="Last updated on Dec 9, 2021, 1:45 PM UTC (59583ae)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-pc/2704/9cb71e8...jan-ivar:59583ae.html" title="Last updated on Dec 9, 2021, 1:45 PM UTC (59583ae)">Diff</a>

### CODE REVIEW

File: webrtc.html [hunk 1]
```diff
@@ -12409,7 +12409,9 @@ <h3>
                     The {{RTCIceTransport}} has finished gathering, received an
                     indication that there are no more remote candidates,
                     finished checking all candidate pairs, and all pairs have
-                    either failed connectivity checks or have lost consent.
+                    either failed connectivity checks or lost consent, and
+                    either zero local candidates were gathered or the PAC timer
+                    has expired [[RFC8863]].
```
Comment (youennf) [inline webrtc.html:12414]: Sounds good, we need to figure out whether revealing PAC timer value is a fingerprinting issue or not.
For instance, can it reveal network type? If a fixed value, seems ok

**Comment by jan-ivar** [comment]
I just noticed Preview doesn't show any of the diagrams, so the modified diagram is:

File: webrtc.html (hunk 1 above)
Comment (jan-ivar) [reply webrtc.html:12414]: [RFC8864 section 4](https://www.rfc-editor.org/rfc/rfc8863.html#section-4) says _"the ICE agent will start its timer once it believes ICE connectivity checks are starting,"_ and _"The RECOMMENDED duration for the PAC timer is equal to the agent's connectivity check transaction timeout, including all retransmissions"_, so this seems network agnostic to me.

Comment (juberti) [reply webrtc.html:12414]: This looks right to me. The PAC timer is meant to be independent of path type, so I agree there shouldn't be a fingerprinting concern.

**Pull request merged**